client/src/components/AddItemFormSteps/StepThree.jsx

```diff
@@ -0,0 +1,197 @@
+import { Link } from "react-router-dom";
+import { useEffect, useState } from "react";
+import {
+  myAccountPath,
+  profilePath,
+  shopProductPath,
+} from "../../utilities/paths";
+import Select from "react-select";
+import { customStyles } from "../../utilities/selectStyle";
+import { countryList } from "../../utilities/countryList";
+import { postProduct } from "../../utilities/productsApi";
+import { useNavigate } from "react-router-dom";
+import moment from "moment";
+import { DATETIME_FORMAT } from "../../utilities/constants";
+import { postImagesToCloudinary } from "../../utilities/cloudinaryApi";
+import LoadingSpinner from "../LoadingSpinner";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+import { useUserContext } from "../../contexts/UserContextProvider";
+
+const StepThree = ({
+  prevStep,
+  productDetails,
+  setProductDetails,
+  handleInputData,
+  images,
+}) => {
+  const { user } = useUserContext();
+  const [isLoading, setIsLoading] = useState(false);
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    setProductDetails({
+      ...productDetails,
+      address: user.address === null ? "" : user.address,
+      email: user.email,
+      phone: user.phone === null ? "" : user.phone,
+      city: user.city === null ? "" : user.city,
+      country: user.country === null ? "" : user.country,
+      zipCode: user.zipCode === null ? "" : user.zipCode,
+    });
+  }, []);
+
+  const handleCountryChange = (selectedOption) => {
+    setProductDetails({ ...productDetails, country: selectedOption.value });
+  };
+
+  const handlePostItem = async () => {
```

Comment:
Perhaps move this method to AddItemPage, that way StepThree would not need to be aware of images. 
And only provide onNext={handlePostItem} to StepThree and call it from here